components/common/successfulModal/ShareSocialsSection.vue

```diff
@@ -4,30 +4,61 @@
       :class="[
         `flex justify-around px-8 items-center w-full gap-6`,
         { '!gap-4': withCopy },
-      ]">
+      ]"
+    >
```

Comment:
this is very strange rule imo

Reply:
it's customizable, but I prefer leave it as it is.

ref:
`⚙️ This rule is included in all of "plugin:vue/vue3-strongly-recommended"...`: https://eslint.vuejs.org/rules/html-closing-bracket-newline